### DISCUSSION TITLE
NodeJS Agent Kafka support

### DISCUSSION THREAD

**Discussion by muratkarakas** [apache/skywalking#12563] [Q&A] [answered]
When I integrated my sample node application(NestJS based) it worked like a charm but could not see any trace or topology link for Kafka. I have used https://github.com/apache/skywalking-nodejs plugin and when I checked did not see any Kafka related feature. Is it unsupported for NodeJS, if so is there a plan for support?

Thanks

**Comment by wu-sheng** [comment, marked as answer]
I think there is no kafka reporter for nodejs agent.
AFAIK, no one is working on this.
